lib/apipie/extractor/writer.rb

```diff
@@ -80,6 +80,12 @@ def ordered_call(call)
           ordered_call[k] = case call[k]
                      when ActiveSupport::HashWithIndifferentAccess
                        convert_file_value(call[k]).to_hash
+                     when String
+                       if k == "request_data"
+                         remove_binary_data(call[k])
```

Comment:
I feel it would be better to check if the data contain something binary and using some placeholder (as we have in convert_file_value in "<FILE CONTENT '#{v.original_filename}'>").